src/views/tables/PositionsTable.tsx

```diff
@@ -298,13 +323,27 @@ export const PositionsTable = ({
   const perpetualMarkets = useSelector(getPerpetualMarkets, shallowEqual) || {};
   const assets = useSelector(getAssets, shallowEqual) || {};
   const openPositions = useSelector(getExistingOpenPositions, shallowEqual) || [];
+  const openOrders = useSelector(getSubaccountOpenOrders, shallowEqual) || [];
+
+  const stopLossOrders: SubaccountOrder[] = [];
+  const takeProfitOrders: SubaccountOrder[] = [];
+
+  openOrders.map((order: SubaccountOrder) => {
+    if (isStopLossOrder(order)) {
+      stopLossOrders.push(order);
+    } else if (isTakeProfitOrder(order)) {
+      takeProfitOrders.push(order);
+    }
+  });
 
   const positionsData = openPositions.map((position: SubaccountPosition) => ({
     tickSizeDecimals: perpetualMarkets?.[position.id]?.configs?.tickSizeDecimals || USD_DECIMALS,
     asset: assets?.[position.assetId],
     oraclePrice: perpetualMarkets?.[position.id]?.oraclePrice,
+    stopLossOrders: stopLossOrders.filter((order) => order.marketId === position.id),
+    takeProfitOrders: takeProfitOrders.filter((order) => order.marketId === position.id),
     ...position,
-  })) as PositionTableRow[];
```

Comment:
was causing typescript errors due to spread operator types not being [picked up](https://github.com/microsoft/TypeScript/issues/32022), short of listing out all 22 properties from `SubaccountPosition`, this seemed like the easiest resolution